src/style/StyledMain.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+const StyledMain = styled.main`
+  form {
+    display: grid;
+    text-align: center;
+    gap: 1rem;
+  }
+  h2 {
+    display: flex;
+    justify-content: center;
+  }
+  select {
+    width: 11rem;
+    height: 1.5rem;
+    margin: auto;
+  }
+  .workout-plan {
```

Comment:
I recommend that to not work with classes, but with styled compoments for these elements.